src/universal/containers/app/MyAccountPage/MyAccountPage.scss

```diff
@@ -0,0 +1,7 @@
+@import "../../../theme/variables.scss";
+
+.MyAccountPage {
+  width: 600px;
+  margin: 0 auto;
```

Comment:
I'm not really sure if the form should be middle aligned, let's see how it looks left aligned and remove this line :).